frontend/src/CommitInfo.res

```diff
@@ -40,14 +40,16 @@ let renderExternalLink = (~href, text) => {
   <a target="_blank" className={Sx.make(sx)} href> {Rx.text(text)} </a>
 }
 
+let url: string = %raw(`import.meta.env.VITE_FRONTEND_URL`)
+
 let renderJobIdLink = jobId => {
   let shortJobId = switch String.split_on_char('/', jobId) {
   | list{_, shortJobId} => shortJobId
   | _ =>
     Js.log(("Error: invalid jobId", jobId))
     jobId
   }
-  let href = "http://autumn.ocamllabs.io:8081/job/" ++ jobId
+  let href = url ++ ":8081/job/" ++ jobId
```

Comment:
Would that make sense to use another env variable for the port number? So that we can use it in both places (here and in `docker-compose-yaml`)?